app/controllers/idv/getting_started_controller.rb

```diff
@@ -20,7 +20,6 @@ def show
     end
 
     def update
-      flow_session[:skip_upload_step] = true unless FeatureManagement.idv_allow_hybrid_flow?
```

Comment:
This logic has moved to HybridHandoffController, which will redirect straight to document capture if the hybrid flow is unavailable.